services/pgnotifier/pgnotifier.go

```diff
@@ -93,6 +88,11 @@ func loadPGNotifierConfig() {
 	pgNotifierDBport, _ = strconv.Atoi(config.GetEnv("PGNOTIFIER_DB_PORT", "5432"))
 	pgNotifierDBpassword = config.GetEnv("PGNOTIFIER_DB_PASSWORD", "ubuntu") // Reading secrets from
 	pgNotifierDBsslmode = config.GetEnv("PGNOTIFIER_DB_SSL_MODE", "disable")
+	maxAttempt = config.GetInt("PgNotifier.maxAttempt", 3)
```

Comment:
Why did we move these?

Reply:
I moved it because loadPGNotifierConfig would be called anyways and felt it would be better if all config variables are at one place